charts/newrelic-infra-operator/README.md

```diff
@@ -0,0 +1,133 @@
+| `config.infraAgentInjection.agentConfig.podSecurityContext.runAsGroup`          | runAsGroup UID for Pod Security Context.                                                                                                                                                                                                                                                                                                    |                                                                                                      |
+| `config.infraAgentInjection.agentConfig.configSelectors[]`                      | ConfigSelectors is the way to configure resource requirements and extra envVars of the injected sidecar container. When mutating it will be applied the first configuration having the labelSelector matching with the mutating pod.                                                                                                        |                                                                                                      |
+| `config.infraAgentInjection.agentConfig.configSelectors[].resourceRequirements` | ResourceRequirements to apply to the sidecar.                                                                                                                                                                                                                                                                                               |                                                                                                      |
+| `config.infraAgentInjection.agentConfig.configSelectors[].extraEnvVars`         | ExtraEnvVars to pass to the injected sidecar.                                                                                                                                                                                                                                                                                               |                                                                                                      |
+| `config.infraAgentInjection.agentConfig.configSelectors[].labelSelector`        | LabelSelector matching the labels of the mutating pods.                                                                                                                                                                                                                                                                                     |                                                                                                      |
+
+
+
+## Example
+
+Make sure you have [added the New Relic chart repository.](../../README.md#install)
+
+Then, to install this chart, run the following command:
+
+```sh
+helm upgrade --install newrelic/newrelic-infra-operator --set cluster=my_cluster_name
```

Comment:
License is also required, right?

Reply:
yes I am adding it, it was also missing from the readme

Reply:
as well as "cluster"

Reply:
also missing release name from the command